policyengine_us/parameters/gov/states/mt/tax/income/credits/ctc/base.yaml

```diff
@@ -0,0 +1,10 @@
+description: Montana provides the following base amount per child under the child tax credit.
+values:
+  2023-01-01: 1_200
+metadata:
+  units: currency-USD
+  period: year
+  label: Montana child tax credit base
+  reference:
+    - title: 2023 Montana Legislature, House Bill No. 268, Section 1, 7 (c)
```

Comment:
please add a reference to the legal code (which differs from legislation)

also include references to tax forms

Reply:
it seems like the bill has not been implemented yet, but the structure is fully in place now, should make this a reform PR?